pkg/cluster/spec/monitoring.go

```diff
@@ -288,6 +289,13 @@ func (i *MonitorInstance) InitConfig(
 
 		AdditionalArgs: spec.AdditionalArgs,
 	}
+	// Set retention policy
+	if spec.RetentionTime == "" {
```

Comment:
This is to handle the deprecated `Retention`, right? Maybe add a comment about this. Could also be a good place to emit a warning about the use of deprecated things.